flight/Modules/Attitude/attitude.c

```diff
@@ -154,6 +154,8 @@ static const float zeros[3] = {0.0f, 0.0f, 0.0f};
 static struct complementary_filter_state complementary_filter_state;
 static struct cfvert cfvert; //!< State information for vertical filter
 
+static float T[3];
```

Comment:
As long as you're moving `T` around, you might consider renaming the transformation with a more searchable, i.e. distinct, name.